docs/root/start/building.rst

```diff
@@ -6,15 +6,15 @@ Building
 
 The Envoy build system uses `Bazel <https://bazel.build/>`_.
 
-In order to ease initial building and for a quick start, we provide an Ubuntu 16 based docker container that
-has everything needed inside of it to build and *statically link* Envoy, see :repo:`ci/README.md`.
+In order to ease initial building and for a quick start, we provide an Ubuntu 16 and a Windows based docker container
+that has everything needed inside of it to build and *statically link* Envoy, see :repo:`ci/README.md`.
```

Comment:
Small ask for the plural here, 
...'docker containers that have'...

Reply:
sure fixed